fastlane/lanes/release.rb

```diff
@@ -0,0 +1,225 @@
+# frozen_string_literal: true
+
+# Lanes related to the Release Process (Code Freeze, Betas, Final Build, AppStore Submission…)
+#
+platform :ios do
+
+  # Executes the initial steps needed during code freeze
+  # - Cuts a new release branch
+  # - Extract the Release Notes
+  # - Freezes the GitHub milestone and enable GitHub branch protection for the new branch
```

Comment:
Just a punctuation and verb tense consistency nitpick:

```suggestion
  # Executes the initial steps needed during code freeze:
  # - Cuts a new release branch
  # - Extracts the Release Notes
  # - Freezes the GitHub milestone and enable GitHub branch protection for the new branch
```

Alternatively:

```suggestion
  # Executes the initial steps needed during code freeze:
  #
  # - Cuts a new release branch
  # - Extracts the Release Notes
  # - Freezes the GitHub milestone and enable GitHub branch protection for the new branch
```

I prefer the new line before the list starts, but I appreciate the value in keeping the comment compact.